docs/self-managed/concepts/pvc-resize-guide.md

```diff
@@ -0,0 +1,163 @@
+# 🔁 PVC Expansion or Migration Guide
+## ✅ Approach 1: Direct PVC Expansion (if allowVolumeExpansion: true)
+
+### Step 0: Check if Your StorageClass Supports Expansion
+```bash
+kubectl get storageclass -o yaml | grep allowVolumeExpansion
+```
+
+Or for a specific class:
+
+```bash
+kubectl get storageclass <your-storage-class> -o yaml | grep -A5 -B5 allowVolumeExpansion
+```
+
+### Step 1: Check Current PVCs
+```bash
+kubectl get pvc -l app.kubernetes.io/component=data -o wide
```

Comment:
not generally applicable.
This assumes that you have split up your Elasticsearch setup into their distinctive types à la "master", "data", "cooridnator", ... nodes.

E.g. the Camunda Helm chart only uses master nodes that contain the other roles as well, therefore it's just `app.kubernetes.io/component=master`.
So again, highly dependent on the setup of a user.